packages/server/scripts/db-setup.sh

```diff
@@ -4,7 +4,6 @@
 db_result=0
 while [ "${db_result}" -ne "1" ]; do
     echo "Waiting for DB..."
-    # npx sequelize db:migrate:status 2>&1 | grep 'connect ECONNREFUSED' > /dev/null
     npx sequelize db:migrate:status 2>&1 | grep 'connect ECONNREFUSED'
```

Comment:
I'm now realizing that I had this for testing, but I would remove this one and keep the commented out one. This way we don't see that error message in the output while waiting for DB, which is normal but could be misleading (plus we already have the "Waiting for DB..." one above).